### PR TITLE
Feature/template lib 2

### PR DESCRIPTION
**Pull request by RV-Adrian**

### CODE REVIEW

**Comment by RV-Adrian** [comment]
@ghislaineguerin 
Hi Ghislaine, pull request as mentioned yesterday. All details in this video
https://drive.google.com/a/risevision.com/file/d/0B8XS4J7-uR-eN2dlLTM3Q3hURlE/view?usp=sharing

Thanks!

**Comment by ghislaineguerin** [comment]
@RV-Adrian looks good, I've added my comments.

File: src/ui-components/template-picker.scss [hunk 1]
```diff
@@ -173,3 +184,26 @@
...
+    text-align: center;
+    // height: 412px;
+    display: block;
+    & img {
+        width: 60%;
+        text-align: center;
+        margin: 0 auto;
+    }
+}
+#productDetailsModal  {
```
Comment (RV-Adrian) [inline src/ui-components/template-picker.scss]: @ghislaineguerin Good catch. At the time, this was to accomodate for the 'Template Library' experiment's banner without shifting too much. But I see now I can use -btn-multi-line instead. Will update.

File: src/bootstrap-overwrites/_buttons.scss [hunk 2]
```diff
@@ -58,4 +58,25 @@
             margin-bottom: 10px;
         }
     }
-}
\ No newline at end of file
+}
+
+.btn-promo {
```
Comment (RV-Adrian) [inline src/bootstrap-overwrites/_buttons.scss]: @ghislaineguerin I do like .btn-multi-line but do you think it's ok that it also targets ul>li siblings? The design is a 'promotional' type of component.

File: src/app.scss [hunk 3]
```diff
@@ -46,6 +46,7 @@
    ========================================================================== */
 @import "ui-components/modals";
 @import "ui-components/alignment-component";
+@import "ui-components/alignment";
```
Comment (RV-Adrian) [inline src/app.scss]: @ghislaineguerin It should be. I think it this SCSS got axed on the CommStyle overhaul, but noticed there is no replacement in utilities.scss. Would like to add it there in a future PR if that's ok (updating all class name usage at the same time)

Comment (ghislaineguerin) [inline src/app.scss]: @RV-Adrian got it. thanks

File: src/bootstrap-overwrites/_buttons.scss (hunk 2 above)
Comment (ghislaineguerin) [inline src/bootstrap-overwrites/_buttons.scss]: @RV-Adrian I don't get it. why does it target lists?

File: src/ui-components/template-picker.scss (hunk 1 above)
Comment (ghislaineguerin) [inline src/ui-components/template-picker.scss]: @RV-Adrian thanks!

**Pull request merged**